templates/email/generate.py

```diff
@@ -0,0 +1,72 @@
+import os
+import re
+
+
+messages_path = os.path.join(os.getcwd(), 'messages')
+generated_path = os.path.join(os.getcwd(), 'generated')
+template_file_extension = '.html'
+
+
+def get_template_for_message(filename):
+    lang = filename.replace(template_file_extension, '')[-2:]
+    if lang != 'fi' and lang != 'sv' and lang != 'en':
+        lang = 'fi'
+    template_filename = 'base_template_' + lang + template_file_extension
+    template_path = os.path.join(os.getcwd(), template_filename)
+    return open(template_path, 'r').read()
+
+
+def get_content_tag_line(template):
+    return re.search('^.*{{ content }}.*$', template, re.MULTILINE).group(0)
+
+
+def get_indentation(line):
+    return len(line.split('{', 1)[0])
+
+
+def indent_content(content, indentation):
+    content_lines = content.splitlines()
+    indented_content = ''
+    for line in content_lines:
+        if len(line) > 0:
+            indented_content = indented_content + indentation * ' ' + line + '\n'
+        # if line is empty, don't indent
+        else:
+            indented_content = indented_content + '\n'
+    return indented_content
+
+
+def generate_template(filename):
+    message = open(os.path.join(messages_path, filename), 'r').read()
+
+    template = get_template_for_message(filename)
+    content_tag_line = get_content_tag_line(template)
+    indentation = get_indentation(content_tag_line)
+    indented_message_content = indent_content(message, indentation)
```

Comment:
To avoid manually having to close the file, you can do:
```suggestion
    with open(os.path.join(messages_path, filename), 'r') as template_file:
        message = template_file.read()

        template = get_template_for_message(filename)
        content_tag_line = get_content_tag_line(template)
        indentation = get_indentation(content_tag_line)
        indented_message_content = indent_content(message, indentation)
```

Reply:
Great 👍